### DISCUSSION TITLE
How to Flip board orientation

### DISCUSSION THREAD

**Discussion by sbernsto** [mliebelt/pgn-viewer#554] [General]
Is there a way for the PGN-viewer to read a pgn and flip the board?  someone on a stack exchanged mentioned [StartFlipped "1"] but that doesn't work with pgn-viewer.  any other suggestions?

**Comment by mliebelt** [comment]
Please have a look at the [API of pgn-viewer](https://mliebelt.github.io/pgn-viewer/docu.html). The attribute is named `orientation` and allows as values `white` and `black`, default is `white`. This should be part of the configuration, not part of the pgn you want to render. The PGN spec has no tag for orientation. You will find examples how to use it here: https://mliebelt.github.io/pgn-viewer/examples.html#1023

The configuration works for all modes besides `print`.
